src/test/java/com/puppycrawl/tools/checkstyle/utils/CheckUtilTest.java

```diff
@@ -460,6 +407,19 @@ private DetailAST getNodeFromFile(int type) throws Exception {
             JavaParser.Options.WITH_COMMENTS), type);
     }
 
+    /**
+     * Retrieves the AST node from a specific file based on the specified token type.
+     *
+     * @param type The token type to search for in the file.
+     *             This parameter determines the type of AST node to retrieve.
+     * @param file The file from which the AST node should be retrieved.
+     * @return The AST node associated with the specified token type from the given file.
+     * @throws Exception If there's an issue reading or parsing the file.
+     */
+    public static DetailAST getNodeFromSpecificFile(int type, File file) throws Exception {
```

Comment:
make like:
`public static DetailAST getNode(File file, int type)`

Reply:
Not done.

@relentless-pursuit , please do not request review, and just reply to each and every review comment as "done" to confirm you saw it and pushed it

Reply:
Hi @romani. I will keep that in mind. Maybe, I was in a rush to merge the PR. 

Anyways, I assume you asked me to change the file name from `getNodeFromSpecificFile(int type, File file)` to `getNode(File file, int type)`. I have done that possibly. Please do let me know if i understood you right. And, the recent push is available for your review.

Reply:
All good. Thank a lot for fix.